linux_x86_64/include/config.h.in

```diff
@@ -36,7 +36,7 @@
 
 /* Define to 1 if translation of program messages to the user's native
    language is requested. */
-#ifdef __APPLE__
+#if defined __APPLE__ || defined __FreeBSD__
```

Comment:
We don't need any localization of messages at all.
It is "anti-feature" for us. (if I understand correctly what it is about.)

Maybe simply
```
#define ENABLE_NLS 0
```
unconditionally?

Reply:
why not.. pull updated.